web/server/routers/post.ts

```diff
@@ -61,6 +62,19 @@ const postSchema = z.object({
   ),
 });
 
+const postFilterSchema = z.object({
+  type: z.array(z.nativeEnum(FosterType)),
+  breed: z.array(z.nativeEnum(Breed)),
+  age: z.array(z.nativeEnum(Age)),
+  size: z.array(z.nativeEnum(Size)),
+  gender: z.array(z.nativeEnum(Gender)),
+  goodWith: z.array(z.nativeEnum(GoodWith)),
+  behavioral: z.array(z.nativeEnum(Behavioral)),
+  houseTrained: z.nativeEnum(Trained).optional(),
+  crateTrained: z.nativeEnum(Trained).optional(),
```

Comment:
Had a conversation with the LT and we'll be removing crate trained altogether

Reply:
Should I remove it from both then and push that up?

Reply:
Remove it from the filter API route and the filter schema!

Reply:
Should be good now